crates/gpui/src/geometry.rs

```diff
@@ -65,6 +68,8 @@ pub struct Point<T: Default + Clone + Debug> {
     pub y: T,
 }
 
+unsafe impl<T: Copy + Default + Debug + Zeroable + 'static> Pod for Point<T> {}
```

Comment:
Reading through the safety details for bytemuck, I don't think we can safely make this claim, specifically because of the padding bytes restriction here: https://docs.rs/bytemuck/latest/bytemuck/trait.Pod.html. If we had a type such that there was padding at the beginning of the struct, and we stored two of these in a point, we would violate the 'no padding in the middle of the struct' rule. And given [this example](https://stackoverflow.com/q/70587534), it is possible for rust to generate structs which match this definition.